CODEOWNERS

```diff
@@ -31,6 +31,9 @@
 # @trvachov - Timur Rvachov
 # @yzhang123 - Yang Zhang
 
+# TIER 2 Approvers do not modify below
+# +2 Reviewers (Global Tier)
+* @jstjohn @pstjohn @cspades # tier2
```

Comment:
in CODEOWNERS these get overwritten by subsequent rules, right? I.e., 
```
* @pstjohn
docs/ @jwilber 
```
means that only jwilber is the docs owners